### DISCUSSION TITLE
KC 25 - How to experiment the new feature OID4VC

### DISCUSSION THREAD

**Discussion by ahcamy2021** [keycloak/keycloak#30773] [Q&A]
I noticed that you delivered in KC 25 some functionnalites related to OID4VC. I understand that is for experimentation, I want to try it, but, I didn't find documentation explain how to proceed.

Thanks in andance.

**Comment by asvetin** [comment]
I had the same problem, but found this repository that will help you how to test it.
https://github.com/adorsys/keycloak-ssi-deployment
There is no UI for this.
